app/commands.py

```diff
@@ -589,6 +589,17 @@ def process_row_from_job(job_id, job_row_number):
             )
 
 
+@notify_command(name="download-csv-file-by-name")
+@click.option("-f", "--csv_filename", required=True, help="csv file name")
+def download_csv_file_by_name(csv_filename):
+
+    bucket_name = getenv("CSV_BUCKET_NAME")
+    access_key = getenv("CSV_AWS_ACCESS_KEY_ID")
+    secret = getenv("CSV_AWS_SECRET_ACCESS_KEY")
+    region = getenv("CSV_AWS_REGION")
```

Comment:
Are these environment variables the ones that come from our own `.env` files?  If so, they are not available in the cloud.gov environment so this won't work in a `cf run-task` context.  These should be coming from the application configuration instead, which takes care of retrieving the correct environment variables regardless of the environment.

You can see how these are referenced in the [`app/aws/s3.py`](https://github.com/GSA/notifications-api/blob/main/app/aws/s3.py#L61-L68) file, specifically the `get_job_location` method.

In fact, should we be using something like `get_s3_object` from that `aws/s3.py` file in place here?

Reply:
we are calling s3.get_s3_file() which in turn calls s3.get_s3_object()

Reply:
D'oh, you're right!  My mistake there, I mis-read the line!